### DISCUSSION TITLE
Welcome to AdNauseam Discussions !

### DISCUSSION THREAD

**Discussion by dhowe** [dhowe/AdNauseam#1777] [General]
## 👋 Welcome to AdNauseam !
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by McBobb** [comment]
Thanks for making this addon. It's extremely helpful in many ways, but I have to admit that it has a HUGE learning curve.

I came here looking for answers that aren't in the offical Q&A

**Comment by dhowe** [comment]
Happy to help! There are a number of useful documents, including an FAQ linked [here](https://github.com/dhowe/AdNauseam/wiki), or just post your questions below, and someone will reply soon...
